src/components/Routes/Track/Results/ResultsChart/ResultsChart.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { ResponsivePie } from '@nivo/pie';
+import { turnDashesIntoSpacesAndCapitalize } from '../../../../../shared/utilities';
+
+const ResultsChart = (props) => {
+  const { savedApplications } = props;
+
+  const values = savedApplications.reduce((total, current) => {
+    total[current.result] = (total[current.result] || 0) + 1;
+
+    return total;
+  }, {});
+
+  const data = [];
+  for(let value in values) {
+    const name = value === '' ? 'None' : turnDashesIntoSpacesAndCapitalize(value);
+
+    data.push({
+      id: name,
+      label: name,
+      value: values[value]
+    });
+  }
+
+  const rawData = [];
+  savedApplications.forEach((savedApplication) => {
+    rawData.push(savedApplication.result);
+  });
+
+  const fill = [];
+  for(let data in rawData) {
```

Comment:
Seeing as `rawData` is an array, we probably don't want to use a `for...in` to iterate. We could use a `for...of`, but I'd recommend another map!

```
const fill = rawData.map(
  (rawDatum) => ({
    match: {
      id: rawDatum === '' ? 'None' : turnDashesIntoSpacesAndCapitalize(rawDatum)
    },
    id: 'lines'
  })
)
```